packages/hotdog/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "hotdog"
+version = "0.1.0"
+edition = "2018"
+publish = false
+build = "build.rs"
+
+[lib]
+path = "pkg.rs"
+
+[[package.metadata.build-package.external-files]]
+url = "https://github.com/bottlerocket-os/hotdog/archive/c212cfb564850b48aa6008563074a329cee041a1/hotdog-c212cfb.tar.gz"
+sha512 = "68bbd4c4fda72065eae995fe75f0f8d0a30228476051906da4bce3f66352138356bb4dd4fd8e0aa6f5822388f9c7dc04e85fa35038e91be294e4fdd4a6e970bc"
```

Comment:
We want https://github.com/bottlerocket-os/hotdog/commit/f300fb78 for the verbosity fix.